app/components/Dashboard/Table.tsx

```diff
@@ -62,8 +62,12 @@ const Table = ({ applications }: Props) => {
   const applicationNodes = applications.allApplications.nodes;
   const router = useRouter();
 
+  const formPages = Object.keys(schema({}).properties);
+
+  const reviewPage = formPages.indexOf('review') + 1;
```

Comment:
@VB-at-Bis Ok I am getting the review page here now instead of using number or const.